AVRIoT.X/mcc_generated_files/credentials_storage/credentials_storage.h

```diff
@@ -29,11 +29,12 @@
 #ifndef CREDENTIALS_STORAGE_H
 #define CREDENTIALS_STORAGE_H
 
-#define MAX_WIFI_CREDENTIALS_LENGTH 31
```

Comment:
More duplication (in fact incorrect) of symbols to be avoided. Include instead the m2m_types.h file that contains the correct symbols (M2M_MAX_*_LEN)